lib/chef/knife/clc_server_create.rb

```diff
@@ -263,7 +272,7 @@ def check_bootstrap_node_connectivity_params
       end
 
       def check_bootstrap_connectivity_params
-        return if indirect_bootstrap?
+        return if indirect_bootstrap? || config[:bootstrap_protocol] == 'winrm'
```

Comment:
I'm not sure that this is the right course of action. This method checks if we will be able to reach the machine from our client. It checks if we've requested public IP with ssh port open and fails if we didn't. You need to do the same for WinRM.